src/structure/structure.ts

```diff
@@ -0,0 +1,22 @@
+import { apiClient } from '../erudio';
+
+export interface Options {
+  filter: string;
+  page: number;
+  limit: number;
+  sort: string;
+}
+
+export class Structure {
+  private base_url = 'http://edtech-structure-service.dev.svc.cluster.local';
+
+  public async getData(namespace: string, options: Options): Promise<any> {
```

Comment:
@JERILJOSE96 the name is too generic, how would we know it's for fetching a list of structures and not a single one?